tutorials/pl/basic-snap-usage-pl/basic-snap-usage.md

```diff
@@ -0,0 +1,329 @@
+$ sudo zypper install snapd
+```
+
+**OpenWrt**
+
+Aktywuj [snap-openwrt feed].
+
+**Ubuntu**
+W wersji 16.04 LTS desktop wszystko powinno być domyślnie aktywne. Na 14.04, wykonaj instalację poleceniem:
+
+```bash
+$ sudo apt install snapd
+```
+
+Ponieważ wszystko jest gotowe, zainstalujmy pierwszego snapa!
+
+negative
+: Ważne: obecnie wyłączyliśmy SELinux na niektórych dystrybucjach, ponieważ nie dostarcza ono pełnej izolacji i bezpieczeństwa na tych platformach. Ciężko pracujemy, aby wszystkie dystrybucje Linuxa mogły czerpać z tych możliwości jak najszybciej.
+
+
+## Instalacja i uruchamianie pierwszego snapa
+Duration: 3:00
+
+### Wyszukiwanie snapów
+
+Usługa `snapd` jest uruchomiona, więc rozpocznijmy korzystanie z niej! W ten sposób odnajdziesz aplikacje typu „hello world” w sklepie:
+
+```bash
+$ snap find hello
+hello-node-snap       1.0.2        bhdouglass      -  	A simple hello world command
+hello-mdeslaur        2.10 	mdeslaur	-  	GNU Hello, the "hello world" snap
+hello-snap    	0.01 	muhammad	-  	GNU hello-snap, the "Hello, Snap!" snap
+hello         	2.10 	canonical       -  	GNU Hello, the "hello world" snap
+hello-world   	6.3  	canonical       -  	The 'hello-world' of snaps
+hello-sergiusens      1.0  	sergiusens      -  	hello world example
+hello-gabriell	0.1  	gabriell	-  	Qt Hello World example
+hello-bluet   	0.1  	bluet   	-  	Qt Hello World example
+so-hello-world	0.2  	shadowen	-  	the old classic
+hello-huge    	1.0  	noise   	-  	a really big snap
+```
+
+`snap find <wyszukiwana fraza>` przeszuka sklep i zwróci wyniki wraz z wersją pakietu, nazwami autorów i opisem.
+
+### Instalacja i uruchomienie
+
+Jak możesz zauważyć, wielu twórców udostępniło snapy typu „hello world”. Zainstalujmy jeden z nich.
+
+```bash
+$ sudo snap install hello
+
+hello (stable) 2.10 from 'canonical' installed
+```
+
+positive
+: Informacja: kiedy zainstalujesz pierwszego snapa, snapd pobierze postawowy podsystem snapa (który zawiera niewielką liczbę aplikacji o wadze do kilku megabajtów). W rezultacie, czas pobrania pierwszego snapa może być trochę dłuższy niż w przypadku kolejnych.
+
+
+Aby uruchomić pakiet, wprowadź hello w wiersz poleceń (jest to polecenie dostarczone przez snap `hello`:
+
+```bash
+$ hello
+Hello, world!
+```
+
+Oczywiście to tylko prosty przykład, który ma cechy wspólne dla wszystkich snapów, więc jest odizolowany i ma dostęp tylko do własnych danych.
+
+### Sprawdź, które snapy są zainstalowane
+
+Aby wyświetlić listę zainstalowanych snapów, wprowadź `snap list`, które poda również wersję programu, autora i dodatkowe informacje.
+
+
+```bash
+$ snap list
+Name             	Version	Rev  Developer   Notes
+hello            	2.10   	20   canonical   -
+core      	16.04.1	423  canonical   -
+```
+
+Zauważ, że pakiet `core` zawierający podstawowy podsystem również znajduje się na liście. Zawiera on również aktualna wersję snapd, dzięki czemu będzie on zawsze aktualny. 
```

Comment:
Let's reuse the consistent language I suggested earlier.

Reply:
Snapd can only keep itself up-to-date via the `core` snap on certain distributions. This list is altered over time, not sure how to best capture that.